assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSelectionPanel/AudienceItems.js

```diff
@@ -55,35 +55,103 @@ export default function AudienceItems( { savedItemSlugs = [] } ) {
 			return [];
 		}
 
+		// eslint-disable-next-line @wordpress/no-unused-vars-before-return -- We might return before `otherAudiences` is used.
+		const [ siteKitAudiences, otherAudiences ] = audiences.reduce(
+			( [ siteKit, other ], audience ) => {
+				if ( audience.audienceType === 'SITE_KIT_AUDIENCE' ) {
```

Comment:
There's no need to use the `isSiteKitAudience()` selector here, seeing as we have access to `audienceType`. We do compare it directly elsewhere in the codebase, so this is a consistent approach.

Reply:
Nice one 👍